src/block-management/inline-toolbar/index.js

```diff
@@ -52,18 +54,30 @@ export default class InlineToolbar extends React.Component<PropsType> {
 	}
 
 	render() {
+		const { order } = this.props;
+		let moveUpButtonTitle = `Move up from row ${ order }`;
+		if ( this.props.canMoveUp ) {
+			moveUpButtonTitle += ` to row ${ order - 1 }`;
```

Comment:
So, we have learned some bits about best practices for accessibility labels and localization.
In my opinion, this construction for labels looks good, but it's not localized.

Check this for an example how are we localizing accessibility labels:
https://github.com/WordPress/gutenberg/blob/f257ed207fa7cb1dc28e4b1f277af000240fe992/packages/block-library/src/image/edit.native.js#L259-L264

Note the use of `sprintf` for formatted strings, the `/* translators: ...` comment (that will be readable by the translator in GlotPress), and that the whole sentence is written together to give the translator the best context.

Reply:
Yeah thanks @etoledom totally going to do this, I'm having trouble resolving issues with the merged in changes from develop, taking longer than I expected so that's why I haven't pushed that update yet... as soon as I push it I'll ping for the review...

Reply:
@etoledom I made the update in ca0a861, WDYT? 
cc @Tug

Reply:
Hey @JavonDavis !

I think is better to not append strings, since for the translators, the string ` __( ' to row %d' )` will be lacking some context. Beside there are no comments to help them.

I was thinking on something more like this: https://gist.github.com/etoledom/7162a547141490f76c885c0ecd7792ed

This proposed one in the gist looks a bit messy, but we ensure that the whole string gets to the translators and we add a comment that they can read via `/* translators: ...`.

Of course I'm open to suggestions and to improve this proposal too :)

Reply:
Ohh I see, I didn't consider that there'd be more context needed around this... Thanks! Your proposed change I think covers it well

Reply:
@etoledom made that update in dfe8851